assets/javascripts/english-mcas-map.js

```diff
@@ -0,0 +1,104 @@
+  + '</p> <p class="info-window__section"><span class="info-window__category">School Year</span><br />'
+  + data.properties.right_schoolyear
+  + '</p>'
+
+  const windowWithoutData = '<p class="info-window__section"><span class="info-window__category">Municipality (no data)</span><br />' 
+  + data.properties.municipal
+  + '</p>'
+
+  if (data.properties.right_all_pa_p) { return windowWithData }
+  else { return windowWithoutData}
+}
+
+function englishCreateHeatmap(data){
+    const tooltip = d3.select("body").append("div")
+                    .attr("class","tooltip")
+                    .style("opacity",0)
+    const colors = d3.scaleSequentialQuantile()
+                    .interpolator(d3.interpolateRdPu)
+                    .domain(data.features.map(feature => feature.properties.right_all_pa_p))
+    const englishMap = d3.select('.english-mcas-map')
+    const path = d3.geoPath().projection(englishProjection)
+    englishMap.append('g')
+            .attr('class','heatmap')
+            .selectAll('path')
+            .data(data.features)
+            .enter()
+            .append('path')
+            .attr('fill', d => { return colors(d.properties.right_all_pa_p)})
+            .attr('stroke','#ffffff')
+            .attr('stroke-width','1')
+            .attr('stroke-opacity', 0.6)
+            .attr('d', path)
+            .on("mousemove", (d) => {
+                tooltip.transition()
+                .duration(50)
+                .style("opacity", .9);
+                tooltip.html(englishHtmlValue(d))
+                .style("left", (d3.event.pageX + 10) + "px")
+                .style("top", (d3.event.pageY + 10) + "px");
+                }) 
+              .on("mouseleave", d => {
+                tooltip.transition()
+                .duration(200)
+                .style("opacity",0)
+              })
+}
+
+function createBasemap(data) {
+  const tooltip = d3.select("body").append("div")
+                 .attr("class", "tooltip")
+                 .style("opacity", 0);
+
+  const baseMap = d3.select('.english-mcas-map');
+  const path = d3.geoPath().projection(englishProjection);
+  baseMap.append('g')
+    .attr('class', 'heatmap')
+    .selectAll('path')
+    .data(data.features)
+    .enter()
+    .append('path')
+    .attr('fill', '#d9d9d9')
+    .attr('stroke', '#ffffff')
+    .attr('stroke-width', '1')
+    .attr('stroke-opacity', 0.6)
+    .attr('d', path)
+    .on("mousemove", (d) => {
+      tooltip.transition()
+      .duration(50)
+      .style("opacity", .9);
+      tooltip.html(englishHtmlValue(d))
+      .style("left", (d3.event.pageX + 10) + "px")
+      .style("top", (d3.event.pageY + 10) + "px");
+    })
+    .on("mouseleave", d => {
+      tooltip.transition()
+      .duration(200)
+      .style("opacity",0)
+    })
+    
+}
+
+d3.json('/assets/data/base-map.json').then((data) => {
+  createBasemap(data);
+  d3.json('/assets/data/3rd-grade-english-mcas.json').then(data => {
```

Comment:
Instead of nesting you can try the pattern I used on Perfect Fit https://github.com/MAPC/perfect-fit/blob/800b33d6f3ae603e0f29abf430c8eecfc69dc907/assets/javascripts/parking_site_map.js#L298-L337